dom.bs

```diff
@@ -5731,6 +5737,9 @@ It is initially set to false.</p>
 <!-- If we ever change this, e.g., add a ShadowRoot object constructor, that would have serious
      consequences for innerHTML. -->
 
+<p><a for=/>Shadow roots</a> have an associated <dfn for=ShadowRoot>slot assignment</dfn>
+("<code>manual</code>" or "<code>auto</code>"), whose default value is "<code>auto</code>".</p>
```

Comment:
I don't think we have to state a default/initial value as it's always given. This is also a problem with delegates focus.

Reply:
Removed default value.